proc-macros/src/render_server.rs

```diff
@@ -400,7 +406,7 @@ impl RpcDescription {
 							#tracing::error!("Failed to parse JSON-RPC params as object: {}", e);
 							let _e: #err = e.into();
 							#pending.reject(_e);
-							return;
+							return Ok(());
```

Comment:
we need to document that this is a "hack" i.e, the result of this is not used anyway just make the API more ergonomic such that users can just do `params.parse()?`

Reply:
I removed it recently just for that reason :sweat_smile: